judo-ui-react/src/main/resources/actor/src/pages/components/table/for-association.hbs

```diff
@@ -94,7 +94,10 @@ export const {{ tableComponentName table }} = forwardRef<RefreshableTable, {{ ta
                     id: '{{ createId action }}',
                     label: t('judo.pages.table.remove', { defaultValue: 'Remove' }) as string,
                     icon: <MdiIcon path="{{ action.icon.name }}" />,
-                    action: async (row: {{ classDataName table.dataElement.target 'Stored' }}) => {{ actionFunctionName action }}(ownerData, row, () => fetchData()),
+                    action: async (row: {{ classDataName table.dataElement.target 'Stored' }}) => {{ actionFunctionName action }}(ownerData, row, () => {
+                        fetchData();
+                        fetchOwnerData();
```

Comment:
Calling fetchOwnerData() should bump the refresh counter, and after that the grid's refresh should trigger implicitly. Currently if we observe the network tab we will see double calls which is not ok. Please double check this and remove the fetchData call if it is not necessary.

Reply:
[b6f1eb0](https://github.com/BlackBeltTechnology/judo-ui-react-template/pull/202/commits/b6f1eb072e1791a01e72145d30f6a695511dc04c)